docs/index.md

```diff
@@ -73,31 +97,18 @@ That's it! You have now installed DDEV on WSL2. If you're using WSL2 for DDEV (r
 
```

Comment:
This was something I overlooked on the initial PR.  Might want to remove lines 13-16 and mention the install script instead?  We want to remove all traces of Homebrew for both Windows and Linux, right? :)

Reply:
Well *you* want to remove all traces of homebrew... I'm fine with it.

Reply:
```suggestion
- 14. Install Homebrew: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` (See [brew.sh](https://brew.sh/).)
- 15. Add brew to your path as prompted:, `echo 'eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)' >> ~/.profile && source ~/.profile`
- 16. `brew install gcc && brew install drud/ddev/ddev`
```

Reply:
Above line 96:
```suggestion
Follow the instructions in the `Linux, macOS and Windows WSL2 (install script)` section above.
```